### ISSUE TITLE
IEC61512

### ISSUE THREAD

**Issue by markusgraube** [i40-Tools/vocabularies#22]
Hi,

i just pushed an ontology with iec61512 concepts. However, perhaps it should be worth to align it with the best practises you used for the other vocabularies (using skos etc.)

**Comment by igrangel** [comment]
Yes I agree. One important thing is to provide, as much as possible metadata for the concepts. For example, `skos:prefLabel` and `skos:definition` for every class, property, etc.
